samples/projects/java-project/README.md

```diff
@@ -0,0 +1 @@
+This example is fully functional, use `./gradlew build` or `./gradlew run` to see it in action.
```

Comment:
We could just recommend running this the normal gradle way, either `../../../gradlew run` or `./gradlew :samples/projects/java-project:run` (or something like that, didn't actually try it) from the root directory.

Reply:
Unfortunately `./gradlew :samples/projects/java-project:run` doesn't do what you want it to. Running subprojects is a complicated matter; basically the root project needs to define all the included subprojects, even if another subproject defines it in its settings.gradle. So basically we would have to include the subproject in the root settings.gradle and reference `:horseshoe` in the root project as well.